src/templates/Dashboard.vue

```diff
@@ -35,12 +35,14 @@
               </b-input-group>
             </b-form-group>
           </b-col>
+          <div v-if="showAdmin" >
+            <b-button v-on:click="editFacility()">Create Facility</b-button>
```

Comment:
This function seems to be adding the facility more than editing it?

Reply:
yeah it was just a copy/paste of another function from another component. I got lazy and didn't re-name it.